[2024-02-19T17:42:01.530+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: primeira_dag.tsk1 manual__2024-02-19T17:42:01.017061+00:00 [queued]>
[2024-02-19T17:42:01.536+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: primeira_dag.tsk1 manual__2024-02-19T17:42:01.017061+00:00 [queued]>
[2024-02-19T17:42:01.536+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-02-19T17:42:01.536+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-02-19T17:42:01.536+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-02-19T17:42:01.544+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): tsk1> on 2024-02-19 17:42:01.017061+00:00
[2024-02-19T17:42:01.547+0000] {standard_task_runner.py:55} INFO - Started process 910 to run task
[2024-02-19T17:42:01.549+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'primeira_dag', 'tsk1', 'manual__2024-02-19T17:42:01.017061+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/primeiradag.py', '--cfg-path', '/tmp/tmpzfr35clf']
[2024-02-19T17:42:01.549+0000] {standard_task_runner.py:83} INFO - Job 29: Subtask tsk1
[2024-02-19T17:42:01.581+0000] {task_command.py:388} INFO - Running <TaskInstance: primeira_dag.tsk1 manual__2024-02-19T17:42:01.017061+00:00 [running]> on host 62357f961983
[2024-02-19T17:42:01.621+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=primeira_dag
AIRFLOW_CTX_TASK_ID=tsk1
AIRFLOW_CTX_EXECUTION_DATE=2024-02-19T17:42:01.017061+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-19T17:42:01.017061+00:00
[2024-02-19T17:42:01.622+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-02-19T17:42:01.622+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep']
[2024-02-19T17:42:01.628+0000] {subprocess.py:86} INFO - Output:
[2024-02-19T17:42:01.630+0000] {subprocess.py:93} INFO - sleep: missing operand
[2024-02-19T17:42:01.630+0000] {subprocess.py:93} INFO - Try 'sleep --help' for more information.
[2024-02-19T17:42:01.630+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-02-19T17:42:01.635+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-02-19T17:42:01.638+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=primeira_dag, task_id=tsk1, execution_date=20240219T174201, start_date=20240219T174201, end_date=20240219T174201
[2024-02-19T17:42:01.644+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 29 for task tsk1 (Bash command failed. The command returned a non-zero exit code 1.; 910)
[2024-02-19T17:42:01.680+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-02-19T17:42:01.696+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
